### PR TITLE
Fix CIS EKS 3.2.5

### PR DESCRIPTION
**Pull request by orouz**
### Summary of your changes

fail the rule if the value of `--streaming-connection-idle-timeout` or `streamingConnectionIdleTimeout` is `"0s"`, `"0"` or `0` 

| Screenshot 2023-04-23 at 14 00 48 | Screenshot 2023-04-23 at 14 01 14 |
|---------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------|

### Related Issues

- closes: https://github.com/elastic/cloudbeat/issues/632

### Checklist

- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added the necessary README/documentation (if appropriate)

> **Note** Make sure to bump cloudbeat [`version/policy.go`](https://github.com/elastic/cloudbeat/blob/main/version/policy.go#L20) after merging this PR and drafting a new release.

### CODE REVIEW

File: bundle/compliance/cis_eks/rules/cis_3_2_5/rule.rego [hunk 1]
```diff
@@ -9,9 +9,21 @@ rule_evaluation = false {
 	audit.process_contains_key_with_value("--streaming-connection-idle-timeout", "0")
 }
 
+rule_evaluation = false {
+	audit.process_contains_key_with_value("--streaming-connection-idle-timeout", "0s")
+}
```
Comment (orouz) [inline bundle/compliance/cis_eks/rules/cis_3_2_5/rule.rego:14]: process argument can only be a string, so either `0` or `0s` is sufficient

File: bundle/compliance/cis_eks/rules/cis_3_2_5/rule.rego [hunk 2]
```diff
@@ -9,9 +9,21 @@ rule_evaluation = false {
...
 	audit.not_process_arg_comparison("--streaming-connection-idle-timeout", ["streamingConnectionIdleTimeout"], 0)
 }
 
+rule_evaluation = false {
+	audit.not_process_arg_comparison("--streaming-connection-idle-timeout", ["streamingConnectionIdleTimeout"], "0")
+}
+
+rule_evaluation = false {
+	audit.not_process_arg_comparison("--streaming-connection-idle-timeout", ["streamingConnectionIdleTimeout"], "0s")
+}
```
Comment (orouz) [inline bundle/compliance/cis_eks/rules/cis_3_2_5/rule.rego:27]: external config value can technically be `0`, `"0s"` or `"0"`

Comment (uri-weisman) [reply bundle/compliance/cis_eks/rules/cis_3_2_5/rule.rego:27]: I think that the `streamingConnectionIdleTimeout` value coming from [kubeconfig](https://kubernetes.io/docs/reference/config-api/kubelet-config.v1beta1/) can not be an integer.
Its value is [Duration](https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1#Duration) which seems to be marshaled to string. ("In particular, it marshals into strings")

Comment (orouz) [reply bundle/compliance/cis_eks/rules/cis_3_2_5/rule.rego]: removed the `int` case

**Review by uri-weisman — APPROVED** [review]

**Pull request merged**